### PR TITLE
Callibration command

### PR DESCRIPTION
**Pull request by nilzilch**
Added support for calibration, removed data component 'all', rt protocol version to 1.20 and updated version to 2.0.5.

### CODE REVIEW

File: qtm/qrt.py [hunk 1]
```diff
@@ -125,7 +127,7 @@ def has_transport(self):
     async def get_current_frame(self, components=None) -> QRTPacket:
         """Get measured values from QTM for a single frame.
 
-        :param components: A list of components to receive, could be 'all' or any combination of
+        :param components: A list of components to receive, could be any combination of
                 '2d', '2dlin', '3d', '3dres', '3dnolabels',
                 '3dnolabelsres', 'analog', 'analogsingle', 'force', 'forcesingle', '6d', '6dres',
                 '6deuler', '6deulerres', 'gazevector', 'image', 'timecode',
```
Comment (helmesjo) [inline qtm/qrt.py]: `skeleton`?

File: qtm/qrt.py [hunk 2]
```diff
@@ -303,14 +299,31 @@ def has_transport(self):
             timeout=self._timeout,
         )
 
-        # TODO GetCaptureC3D
-        # TODO GetCaptureQTM
+    async def calibrate(self, timeout=600):  # Timeout 10 min.
```
Comment (helmesjo) [inline qtm/qrt.py]: I guess some kind of test should be created for this..?

File: qtm/qrt.py (hunk 1 above)
Comment (helmesjo) [reply qtm/qrt.py]: nvm

File: qtm/qrt.py (hunk 2 above)
Comment (nilzilch) [reply qtm/qrt.py]: I have added tests for calibration command.

**Pull request merged**